FeedStoreChallenge/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly.

It was written explicitly in the first lectures of the program for teaching purposes, but you don't need to do it in your code.

Please, remove all explicit `internal` keywords.
```suggestion
	@NSManaged var timestamp: Date
```